packages/inferno/src/DOM/hydration.ts

```diff
@@ -128,17 +158,33 @@ function hydrateElement(vNode: VNode, dom: Element, lifecycle: LifecycleClass, c
 	return dom;
 }
 
-function hydrateChildren(children: InfernoChildren, parentDom: Element, lifecycle: LifecycleClass, context: Object, isSVG: boolean): void {
+function hydrateChildren(
+	children: InfernoChildren,
+	parentDom: Element,
+	lifecycle: LifecycleClass,
+	context: Object,
+	isSVG: boolean
+): void {
 	normalizeChildNodes(parentDom);
 	let dom = parentDom.firstChild;
 
 	if (isArray(children)) {
-		for (let i = 0, len = (children as Array<string | number | VNode>).length; i < len; i++) {
-			const child = children[ i ];
+		for (
```

Comment:
Some options to avoid making it 80 columns:
- Move the len on its own line
- Name the long type (why do you even need the cast in the first place?!)
- Use for of